lighty-netconf-device/src/main/java/io/lighty/netconf/device/NetconfDeviceBuilder.java

```diff
@@ -125,6 +127,7 @@ public NetconfDeviceBuilder withRpcHandler(RpcHandler rpcHandler) {
     }
 
     public NetconfDeviceBuilder withDefaultNotificationProcessor() {
+        this.withCapabilities(ModelUtils.DEFAULT_NOTIFICATION_CAPABILITY);
```

Comment:
please use consistent approach for setting the capabilities: up there it is set via `allCapabilities.addAll()` here you are using `withCapabilities`